metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/EnaObjectQuery.java

```diff
@@ -16,7 +16,7 @@
  *
  */
 
-package uk.ac.ebi.ampt2d.metadata.importer.objects;
+package uk.ac.ebi.ampt2d.metadata.importer.database;
```

Comment:
Couldn't this be part of the `persistence` package? Maybe as a subpackage `queries` or similar. Otherwise it it confusing why are there `persistence` and `database` packages.

Reply:
there is no more persistence package now, so have put it in database now